dvc/dagascii.py

```diff
@@ -2,8 +2,12 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
+import logging
 import math
+import os
+import pydoc
 import sys
+from distutils.spawn import find_executable
```

Comment:
AFAIK distutils is quite heavy to import, hence why we got rid of it in `setup.py` and are using `fastentrypoints.py`. Seems like we could totally use the same approach pydoc uses and call `os.system` instead. What do you think?

Reply:
> Looks good, please see a few more comments above. slightly_smiling_face

Thanks for the review and all catches.
I like all the comments and i'll update the code soon.
 :+1: